app/shared/models/git_repo.rb

```diff
@@ -271,7 +271,7 @@ def setup_repo
                 begin
                   repo.add(all: true)
                   repo.commit("Sync changes")
-                  git.push(GitRepo::DEFAULT_REMOTE, branch: "master", force: true) unless GitRepo.pushes_disabled?
+                  git.push(GitRepo::DEFAULT_REMOTE, "master", force: true) unless GitRepo.pushes_disabled?
```

Comment:
Have you tried this with a different branch name? Like, just hardcoded "my_branch"?

Reply:
No i haven't. Is there a scenario you're worried about?